index.html

```diff
@@ -91,37 +91,39 @@ <h3 class="blue-fg">LET'S CONNECT</h3>
             <!-- WORK cards section -->
             <section class="grid-box white-bg black-fg" id="work-cards-section" >
 
```

Comment:
- Good job so far 👍 your design looks good and it follows the template but there are still some issues to be addressed.
- [ ] You'll notice that the background color of the first page is white and the bottom-left corner of the first page is curved. Then after the first page the background color changes. It would be great if you could implement this feature.
- Your design:
![Annotation 2022-02-20 065927](https://user-images.githubusercontent.com/85113567/154830972-5ecf8840-8c3b-4a39-94a3-a4f9c496b549.jpg)
- Figma design:
![Annotation 2022-02-20 065949](https://user-images.githubusercontent.com/85113567/154830980-6a6cf1ba-43a2-44a2-9e79-28a10e9707dc.jpg)